cirq/ops/pauli_gates.py

```diff
@@ -119,6 +117,14 @@ def _from_json_dict_(cls, exponent, global_shift, **kwargs):
         assert exponent == 1
         return Pauli._XYZ[0]
 
+    @property
+    def basis(self):
```

Comment:
Missing type annotation (applies to other basis methods, too.)

Reply:
added!